### DISCUSSION TITLE
Welcome to OpenAPI-to-Graphql Discussions!

### DISCUSSION THREAD

**Discussion by Alan-Cha** [IBM/openapi-to-graphql#414] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarcoDaniels** [comment]
Hey there 👋 

Pretty cool project here! Bumped into it while looking for a tool to transform OpenAPI specs into GraphQL schema.

Just as an idea, could we organize the Roadmap #342 within the Projects tab in Github? I believe it would bring a better overview on the stages of development :)

**Comment by S1ngl3-x** [comment]
Hello,
the project really is a life saver. We're using it to create a graphql interface to access old REST APIs and so far it's been great.

I am however uncertain about format to scalar conversion. Number format:int64 gets converted to custom scalar BigInt (which makes sense) however String format:date does not get converted to custom scalar Date or something similar.

Is this is a bug or not supported on purpose? If this is not supported is there an option to specify format-> custom scalar conversion?
